### PR TITLE
feat: Nuxt ^2.3.x compatibility

### PR DESCRIPTION
**Pull request by paulgv**
This brings a small change to the module's code to make it compatible with Nuxt v2.3.x and its lerna
setup
Additionally, the package has been refactored:
- Remove nuxt-module-builder dependency as it has been deprecated
- Update ESLint config to use @nuxtjs/eslint-config
- Remove obsolete build step since Nuxt now supports ES6 out of the box
- Simplify files structure (move index.js at the root, remove template/ directory)
- Use Yarn only, drop NPM

BREAKING CHANGE: Drops compatibility with Nuxt v2.2.x and earlier

### CODE REVIEW

**Comment by pi0** [comment]
@paulgv Thanks for this PR. The release script should be executed at the master branch. No need to be included in the PR :)

File: package.json [hunk 1]
```diff
@@ -1,48 +1,46 @@
 {
-  "name": "@nuxtjs/router",
-  "version": "1.1.0",
+  "name": "@ax2/router",
```
Comment (pi0) [inline package.json]: Typo here?

Comment (paulgv) [reply package.json]: Wow sorry about that @pi0! I needed to publish the package under our own scope and I pushed it to the master branch... silly me! Let me redo the PR

Comment (paulgv) [reply package.json]: There you go: https://github.com/nuxt-community/router-module/pull/25

**Comment by paulgv** [comment]
Closing in favor of https://github.com/nuxt-community/router-module/pull/25

**Pull request closed without merging**